integration-tests/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "jest-junit": {
     "suiteName": "Integration tests",
-    "addFileAttribute": true,
+    "addFileAttribute": "true",
```

Comment:
This doesn't seem right. Does `jest-junit` really want the string `"true"`?

Reply:
Stunningly, yes. https://github.com/jest-community/jest-junit#configuration specifies that it must be a string for this particular config key. I think it's because it's being passed through to some underlying thing.